apps/ehr/src/telemed/features/appointment/MedicalHistoryTab/CurrentMedications/CurrentMedicationGroup.tsx

```diff
@@ -56,7 +56,8 @@ function CurrentMedicationItem({ value }: { value: MedicationDTO }): JSX.Element
 
   return (
     <Typography variant="body2">
-      {value.name} {`(${value.intakeInfo?.dose} ${lastIntakeDateDisplay})`}
+      {value.name}
+      {(value.intakeInfo?.dose || lastIntakeDateDisplay) && `(${value.intakeInfo?.dose} ${lastIntakeDateDisplay})`}
```

Comment:
This approach still allows one of the values to be undefined, could do this instead:

```suggestion
{[value.name, value.intakeInfo?.dose, lastIntakeDateDisplay].filter(Boolean).join(' ')}
```